### PR TITLE
#1200: fixed and cleaned docker commandlet

### PR DESCRIPTION
**Pull request by hohwille**
fix for #1200

### CODE REVIEW

File: scripts/src/main/resources/scripts/command/docker [hunk 1]
```diff
@@ -50,101 +50,52 @@ function doSetup() {
...
+  then
+    doFail "It seems that you are running the setup from inside a WSL shell.\nPlease use git-bash for that purpose. Otherwise your OS will be interpreted as Linux"
+  fi
   if [ "${DOCKER_EDITION}" != "docker" ]
   then
-    echo "Rancher Desktop set as option"
-    if uname -r | grep -i microsoft
+    DOCKER_EDITION="rancher"
+    # TODO what all is this for? IMHO we can remove the following IF statement and doIsRancherInstalled function
+    if doIsRancherInstalled
```
Comment (CREITZ25) [inline scripts/src/main/resources/scripts/command/docker:61]: I think line 61 to line 65 in not needed

File: scripts/src/main/resources/scripts/command/docker
Comment (CREITZ25) [inline scripts/src/main/resources/scripts/command/docker:1]: Looks good. If it is tested and works we could merge it

**Review by CREITZ25 — APPROVED** [review]

**Comment by hohwille** [comment]
There still is some bug but not from this PR:
```
******** ATTENTION ********
Installation failed from D:\projects\devonfw\updates\install\docker\docker-1.9.1-rancher-windows.msi
We are sorry for the inconvenience. Please check the above errors, resolve them and try again.

$ ls /D/projects/devonfw/updates/install/docker/docker-1.9.1-rancher-windows.msi
/D/projects/devonfw/updates/install/docker/docker-1.9.1-rancher-windows.msi
```

**Comment by hohwille** [comment]
MSI issue is fixed.
I also found another bug that actually more relates to #1229 but I included the fix in this PR via [this commit](https://github.com/devonfw/ide/pull/1214/commits/5260bf0ad78a268a427fadc187417ccb688b222f).
Now ready for merge.

**Pull request merged**